deploy/008-deploy-vaults.ts

```diff
@@ -0,0 +1,64 @@
+import deployedContracts from "@venusprotocol/governance-contracts/deployments/deployments.json";
+import { ethers } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const xvsVaultDeployment = await deploy("XVSVault", {
+    from: deployer,
+    args: [],
+    log: true,
+    autoMine: true,
+  });
+
+  const xvsVaultAddress = xvsVaultDeployment.address;
+
+  const xvsVaultProxyDeployment = await deploy("XVSVaultProxy", {
+    from: deployer,
+    args: [],
+    log: true,
+    autoMine: true,
+  });
+
+  const xvsVaultProxyAddress = xvsVaultProxyDeployment.address;
+
+  await deploy("XVSStore", {
+    from: deployer,
+    args: [],
+    log: true,
+    autoMine: true,
+  });
+
+  const xvs = await ethers.getContract("XVS");
+  const xvsVault = await ethers.getContract("XVSVault");
+  const xvsStore = await ethers.getContract("XVSStore");
+  const xvsVaultProxy = await ethers.getContract("XVSVaultProxy");
+  const chainId = (await hre.getChainId()) as keyof typeof deployedContracts;
+  const accessControlManager = hre.network.live
+    ? await ethers.getContractAt(
+        "AccessControlManager",
+        deployedContracts[chainId][0].contracts.AccessControlManager.address,
+      )
+    : await deployments.get("AccessControlManager");
+
+  let txn = await xvsVault.setXvsStore(xvs.address, xvsStore.address);
+  await txn.wait(1);
+
+  txn = await xvsVault.setAccessControl(accessControlManager.address);
+  await txn.wait(1);
```

Comment:
These should be called on proxy after _become. Otherwise you're setting values (including _notEntered) in the implementation storage which isn't used when you call functions via proxy